geometry_msgs/Point point
---
string place_name
bool done
